### ISSUE TITLE
[OpenApi] Explode attributes in parameters is not right normalized and cannot be overriden

### ISSUE THREAD

**Issue by tyx** [api-platform/core#4124]
**API Platform version(s) affected**: 2.6.x

**Description**  

The new OpenApi implementation is really nice ! 

It just seems the `explode` attributes of `Model\Parameters` is not right normalized : 
https://github.com/api-platform/core/blob/4717bd5643d7761707229ed8705ae6824248f7e3/src/OpenApi/Model/Parameter.php#L101-L104

Replacing by  `getExplode` make it displaying in the swagger ui.

And moreover, it cannot be overriden

https://github.com/api-platform/core/blob/4717bd5643d7761707229ed8705ae6824248f7e3/src/OpenApi/Factory/OpenApiFactory.php#L383-L397

What about something like : 

```php 
$parameters[] = new Model\Parameter(
                    $name,
                    'query',
                    $data['description'] ?? '',
                    $data['required'] ?? false,
                    $data['openapi']['deprecated'] ?? false,
                    $data['openapi']['allowEmptyValue'] ?? true,
                    $schema,
                    'array' === $schema['type'] && \in_array($data['type'],
                    [Type::BUILTIN_TYPE_ARRAY, Type::BUILTIN_TYPE_OBJECT], true) ? 'deepObject' : 'form',
                    $data['openapi']['explode'] ?? ('array' === $schema['type']),
                    $data['openapi']['allowReserved'] ?? false,
                    $data['openapi']['example'] ?? null,
                    isset($data['openapi']['examples']
                ) ? new \ArrayObject($data['openapi']['examples']) : null);
```

**How to reproduce**  

Just drop a filter and watch the swagger-data in the swagger ui.

**Possible Solution**  

Not sure if both suggestion are the best. But if we think they are, I'm open to make a PR.

**Comment by soyuka** [comment]
:+1: on the suggestion but you should reuse the `'array' === $schema['type'] && \in_array($data['type'], 
     [Type::BUILTIN_TYPE_ARRAY, Type::BUILTIN_TYPE_OBJECT], true)` test

**Comment by vincentchalamon** [comment]
Fixed in 2.6.4
